client.go

```diff
@@ -574,6 +605,19 @@ func (s *Client) getExpiry(height int32, terms *LoopOutTerms,
 func (s *Client) LoopOutQuote(ctx context.Context,
 	request *LoopOutQuoteRequest) (*LoopOutQuote, error) {
 
+	if request.AssetRFQRequest != nil {
+		rfqReq := request.AssetRFQRequest
+		if rfqReq.AssetId != nil && rfqReq.AssetEdgeNode == nil {
```

Comment:
If the rfq request is non-nil, we could just check `rfqReq.AssetId == nil || rfqReq.AssetEdgeNode == nil` to error out.